app/assets/stylesheets/widgets/form.scss

```diff
@@ -11,90 +15,95 @@
       font-size: 11px;
   }
 
+  label {
+    display: block;
+    margin: 0px;
+  }
+
   .inputs {
-    display: table;
+    display: table-row;
     width: 100%;
-    border-spacing: 5px;
 
-    .label {
+    .input {
       display: table-cell;
       padding: 5px;
     }
-  }
 
-  label, .input {
-    display: block;
-    margin: 0px;
-  }
-
-  .input > span {
+    .input > span {
       width: 20%;
       padding-right: 10px;
       margin-top: 10px;
       color: #888;
-  }
+    }
 
-  .chosen-container, textarea, select,
-  input[type="search"], input[type="text"], input[type="number"], input[type="password"], input[type="email"],
-  input[type="checkbox"], input[type="radio"], input[type="tel"] {
-    border: 1px solid #DADADA;
-    color: #888;
-    height: 30px;
-    margin-bottom: 16px;
-    margin-right: 9px;
-    margin-top: 2px;
-    outline: 0 none;
-    padding: 3px 3px 3px 5px;
-    width: 100%;
-    font-size: 12px;
-    line-height:15px;
-    box-shadow: inset 0px 1px 4px #ECECEC;
-    -moz-box-shadow: inset 0px 1px 4px #ECECEC;
-    -webkit-box-shadow: inset 0px 1px 4px #ECECEC;
-  }
+    .chosen-container, textarea, select,
+    input[type="search"], input[type="text"], input[type="number"], input[type="password"], input[type="email"],
+    input[type="checkbox"], input[type="radio"], input[type="tel"] {
+      border: 1px solid #DADADA;
+      color: #888;
+      height: 30px;
+      margin-bottom: 16px;
+      margin-right: 9px;
+      margin-top: 2px;
+      outline: 0 none;
+      padding: 3px 3px 3px 5px;
+      width: 100%;
+      font-size: 12px;
+      line-height:15px;
+      box-shadow: inset 0px 1px 4px #ECECEC;
+      -moz-box-shadow: inset 0px 1px 4px #ECECEC;
+      -webkit-box-shadow: inset 0px 1px 4px #ECECEC;
```

Comment:
Color literals like `#ECECEC` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#ECECEC` should be written as `#ececec`<br>`0px` should be written without units as `0`